football-website/src/functions/auth.service.js

```
@@ -26,10 +26,49 @@ const getJwtHeader = () => {
+    try {
+        const user = JSON.parse(localStorage.getItem("user"));
+        return user?.roles?.[0] || user?.role || null;
+    } catch (error) {
+        console.error("Error parsing user from localStorage:", error);
+        return null;
+    }
+};
+
+// football-website/src/functions/auth.service.js
+
+const getCurrentUser = () => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getCurrentUser'. Consider to delete it or rename it.

'getCurrentUser' is defined here:


(lint/suspicious/noRedeclare)